src/components/pages/Offer/Offer/OfferDetails/OfferForm/Messages/Message.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import Icon from 'components/layout/Icon'
+import { CGU_URL } from 'utils/config'
+
+const Message = ({ message, type }) => {
```

Comment:
Petite suggestion parce que par principe je ne suis pas hyper fan des noms de composants aussi standard et vague que ça.  C'est pas non plus hyper précis mais ça permet de savoir ce que tu importes
```suggestion
const FormMessage = ({ message, type }) => {
```